src/common/com/intellij/plugins/haxe/util/HaxeElementGenerator.java

```diff
@@ -121,6 +122,14 @@ public static HaxeImportStatement createImportStatementFromPath(Project myProjec
     return PsiTreeUtil.getChildOfType(dummyFile, HaxeImportStatement.class);
   }
 
+  @Nullable
+  public static HaxePsiToken createEmptyStatement(Project myProject) {
+    final HaxeImportStatement importStatement = createImportStatementFromPath(myProject, "Std");
+    PsiElement last = importStatement.getLastChild();
+    assert last instanceof HaxePsiToken;
+    return last instanceof HaxePsiToken ? (HaxePsiToken)last : null;
+  }
```

Comment:
Like the rest of the generators, this creates a file with a known set of elements and extracts the one we want.